### PR TITLE
Fix CI

### PR DESCRIPTION
**Pull request by zucchini-nlp**
# What does this PR do?

Fixes CI which is currently red on many PRs. Caused by https://github.com/huggingface/transformers/pull/34043 and https://github.com/huggingface/transformers/pull/34026 when not all tests were fetched for run

### CODE REVIEW

**Review by ArthurZucker — APPROVED** [review]
Thanks for the prompt fix!

File: tests/generation/test_flax_utils.py [hunk 1]
```diff
@@ -101,6 +101,10 @@ def test_greedy_generate_pt_fx(self):
             pt_model = pt_model_class(config).eval()
             pt_model = load_flax_weights_in_pytorch_model(pt_model, flax_model.params)
 
+            # Generate max 5 tokens only otherwise seems to be numerical error accumulation
+            pt_model.generation_config.max_length = 5
+            flax_model.generation_config.max_length = 5
```
Comment (ydshieh) [inline tests/generation/test_flax_utils.py:106]: should it be `input_ids.shape[-1] + 5` or using `max_new_tokens` instead?

**Review by ydshieh — APPROVED** [review]
Thank you!

File: tests/generation/test_flax_utils.py (hunk 1 above)
Comment (zucchini-nlp) [reply tests/generation/test_flax_utils.py:106]: yeah, could also be max-new-tokens. Interestingly after the linked PR `max_length` behaves same way as `max_new_tokens`

**Comment by techkang** [comment]
I think the model used for testing is too large, which casued the timeout error. I proposed a new PR: https://github.com/huggingface/transformers/pull/34482 trying to fixed it by using a small model instead.

**Pull request merged**